src/synthesizer/python/mod.rs

```diff
@@ -85,7 +89,6 @@ impl Synthesizer for Python {
             });
             for param in have_default_or_special_type_params {
                 let name = &param.name;
-                // example: AWS::EC2::Image::Id, List<AWS::EC2::VPC::Id>, AWS::SSM::Parameter::Value<List<String>>
```

Comment:
this honestly isn't very helpful, I just decided to remove it

Reply:
I've actually found this comment helpful. Can we please leave it in?

Reply:
oh sure, yeah re-added it